src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs

```diff
@@ -60,7 +60,7 @@ public UdpClient(int port, AddressFamily family)
             // Validate the address family.
             if (family != AddressFamily.InterNetwork && family != AddressFamily.InterNetworkV6)
             {
-                throw new ArgumentException(SR.net_protocol_invalid_family, nameof(family));
+                throw new ArgumentException(SR.Format(SR.net_protocol_invalid_family, family), nameof(family));
```

Comment:
According to .NET Framework, this should be

```C#
throw new ArgumentException(SR.Format(SR.net_protocol_invalid_family, "UDP"), nameof(family));
```

https://referencesource.microsoft.com/#System/net/System/Net/Sockets/UDPClient.cs,50

Reply:
done